kiwi/bootloader/config/grub2.py

```diff
@@ -1441,15 +1441,18 @@ def _fix_grub_loader_entries_boot_cmdline(self):
             for menu_entry_file in glob.iglob(loader_entries_pattern):
                 with open(menu_entry_file) as grub_menu_entry_file:
                     menu_entry = grub_menu_entry_file.read()
+                    # Made append only
                     menu_entry = re.sub(
                         r'options (.*)',
-                        r'options {0}'.format(self.cmdline),
+                        r'options \1 {0}'.format(self.cmdline),
                         menu_entry
                     )
                 with open(menu_entry_file, 'w') as grub_menu_entry_file:
                     grub_menu_entry_file.write(menu_entry)
 
     def _fix_grub_loader_entries_linux_and_initrd_paths(self):
+        # Temporarily nuked
+        return
```

Comment:
All the _fix methods are called in ```bootloader/config/grub2.py``` 

```
# Patch the written grub config file to actually work:
# Unfortunately the grub tooling has several bugs and issues
# which prevents it to work in image build environments.
# More details can be found in the individual methods.
# One fine day the following fix methods can hopefully
# be deleted...
if self.xml_state.build_type.get_overlayroot_write_partition() is not False:
    self._fix_grub_root_device_reference(config_file, boot_options)
    self._fix_grub_loader_entries_boot_cmdline()
    self._fix_grub_loader_entries_linux_and_initrd_paths()
```

I think it would be good if the skipping of them per if ```Defaults.is_ostree()``` can be done at this place in the code

Reply:
Sounds like a good idea. As an alternative, we could also check if the files exists as set in the BLS config and only fix them if they don't?

Reply:
Done